src/main/java/it/uniroma3/mat/extendedset/intset/IntSet.java

```diff
@@ -236,7 +236,7 @@
 	 * An {@link Iterator}-like interface that allows to "skip" some elements of
 	 * the set
 	 */
-	public interface IntIterator {
+	public interface IntIterator extends org.roaringbitmap.IntIterator {

```

Comment:
It allows to avoid wrapping of Concise's iterators into RoaringBitmap's iterator in bytebuffer-collections (which uses RoaringBitmap's iterator as "primary" interface)